### ISSUE TITLE
source highlight style does not extract to a file

### ISSUE THREAD

**Issue by diguage** [owenh000/asciidoctor-multipage#34]
The project is a great extension.

I want to use it to publish my documents. So I tried the extension. I found a problem: source highlight style does not extract to a file; the style of asciidoc is extracted to a file.

**Comment by owenh000** [comment]
Hi @diguage, please provide a test case with the command used and expected output versus actual output.

**Comment by diguage** [comment]
> Hi @diguage, please provide a test case with the command used and expected output versus actual output.

I'm submitting an example to [asciidoctor-maven-examples](https://github.com/asciidoctor/asciidoctor-maven-examples), you can pull the PR to check it, https://github.com/asciidoctor/asciidoctor-maven-examples/pull/134

**Comment by owenh000** [comment]
@diguage, I am not familiar with Maven and am not able to provide support for Maven. It is currently outside the scope of this project.

>please provide a test case with the command used and expected output versus actual output

Can you provide the equivalent asciidoctor command, like one of the ones shown in the [README](https://github.com/owenh000/asciidoctor-multipage/blob/master/README.md)? You can even run it directly from a local clone of the repository—see the [Development](https://github.com/owenh000/asciidoctor-multipage/blob/master/README.md#development) section.

**Comment by diguage** [comment]
OK, Let me create a example：

```
= Example Manual
Doc Writer <doc.writer@example.org>
2014-09-09
:icons: font
:source-highlighter: coderay
:coderay-style: monokai
:coderay-linenums-mode: table
:stylesdir: assets/styles/
:linkcss:

This is a user manual for an example project.

[#introduction]
== Introduction

This project does something.
We just haven't decided what that is yet.

[#source-code]
== Source Code

[source,java]
.Java code from project
----
package com.diguage;

/**
 * @author D瓜哥 · https://www.diguage.com/
 */
public class Main {
    public static void main(String[] args)
        System.out.println("Hello, world");
    }
}
----

This page was built by the following command:

 $ mvn
```

```bash
$ asciidoctor -r asciidoctor-multipage -b multipage_html5 -D . -a toc=left index.adoc

$ tree assets
assets
└── styles
    └── asciidoctor.css
```

If I remove `:stylesdir: assets/styles/` and `:linkcss:`, it is OK.

**Comment by owenh000** [comment]
@diguage, thanks for the example. I was able to reproduce the problem: with the stock html5 converter, Asciidoctor writes out `coderay-asciidoctor.css` but with the multipage converter it does not. This may be related to handling of the `copycss` attribute, which "is set by default unless the safe mode is `secure`" (quoting the [documentation](https://docs.asciidoctor.org/asciidoctor/latest/html-backend/stylesheet-modes/#to-copy-or-not-to-copy)).

It is possible to work around this by explicitly setting the `copycss` attribute, either in the document (`:copycss:`) or as a command line option (`-a copycss`).

**Comment by owenh000** [comment]
As implied in my [previous comment](https://github.com/owenh000/asciidoctor-multipage/issues/34#issuecomment-1150509326) and stated in the [documentation](https://docs.asciidoctor.org/asciidoctor/latest/html-backend/stylesheet-modes/#copy), Asciidoctor is responsible for writing the stylesheet(s), not the extension.

With this asciidoctor-multipage extension, Asciidoctor is only really aware of the top-level page. This page seldom or never requires any highlighting, so the highlighter is never used and indicates that no stylesheet is needed. This is worked around with commit e61c95a13754c1b2cbbc51e8418060626eeea251, included in release v0.0.16.
